test/new-e2e/system-probe/test-runner/main.go

```diff
@@ -193,13 +191,22 @@ func testPass(testConfig *testConfig, props map[string]string) error {
 	}
 
 	for _, testsuite := range testsuites {
-		pkg := pathToPackage(testsuite)
+		pkg, err := filepath.Rel(testConfig.testDirRoot, filepath.Dir(testsuite))
+		if err != nil {
+			return fmt.Errorf("could not get relative path for %s: %w", testsuite, err)
+		}
 		junitfilePrefix := strings.ReplaceAll(pkg, "/", "-")
 		xmlpath := filepath.Join(xmlDir, fmt.Sprintf("%s.xml", junitfilePrefix))
 		jsonpath := filepath.Join(jsonDir, fmt.Sprintf("%s.json", junitfilePrefix))
 		args := buildCommandArgs(pkg, xmlpath, jsonpath, testsuite, testConfig)
 
+		log.Printf("/go/bin/gotestsum %s", strings.Join(args, " "))
 		cmd := exec.Command("/go/bin/gotestsum", args...)
+		baseEnv = append(
+			baseEnv,
+			"DD_SYSTEM_PROBE_BPF_DIR="+filepath.Join(testConfig.testDirRoot, "pkg/ebpf/bytecode/build"),
+			"DD_SERVICE_MONITORING_CONFIG_TLS_JAVA_DIR="+filepath.Join(testConfig.testDirRoot, "pkg/network/protocols/tls/java"),
+		)
```

Comment:
Can you add a newline to the end of the `fmt.Fprintf` line about 10 lines down? Github won't let me comment on it.